app/src/org/commcare/activities/HomeNavDrawerController.java

```diff
@@ -229,8 +229,7 @@ private static int getItemIcon(String id) {
             case SAVED_FORMS_ITEM_ID:
                 return R.drawable.ic_saved_forms_nav_drawer;
             case INCOMPLETE_FORMS_ITEM_ID:
-                //TODO: Need to add an icon for this option
-                return -1;
+                return R.drawable.ic_incomplete_nav_drawer;
```

Comment:
can we not use the existing [home_incomplete.xml](https://github.com/dimagi/commcare-android/blob/master/app/res/drawable/home_incomplete.xml) ?

Reply:
@shubham1g5 I considered that but it would distort the `look and feel`, no? In that mode all, the icons are grey and the `home_incomplete` is coloured and requires a background:
![image](https://github.com/user-attachments/assets/ee112095-d7ab-472b-9421-60df2b0dc055)